### PR TITLE
API should only return measurements without data for each measurement

### LINKED ISSUE #17: `read_measuremnts()` should conform to API definition
`read_measuremnts()` besides timestamps of the measurements and corresponding meta data, retrieves the data for each measurement. This does not correspond to the API definition. The implementation needs to be simplified and user can retrieve the data for each measurement separately using `read_single_measuremnt()`

### PR DESCRIPTION
**Pull request by vnadhan**
### Summary
- Closes #17 
- Removed code that retrieves data for each measurement
- Fixed the API description a bit
- Verified the test cases that use the API `read_measurements`

TODO: Update the example ipynbs in the docs folder

### CODE REVIEW

File: mvg/mvg.py [hunk 1]
```diff
@@ -396,7 +396,7 @@ def create_measurement(
 
     # in example
     def read_measurements(self, sid: str):
-        """Retrieves all measurements for all timestamps for a source.
+        """Retrieves all measurements (all timestamps) for a source.
```
Comment (SergioMDCB) [inline mvg/mvg.py:399]: Would not be a good idea just to add in the function description "...(all timestamps and metadata)..."

**Review by SergioMDCB — COMMENTED** [review]
This PR removes the call for the entire timeseries for each measurement.
Just a minor comment/update on the function description for clarity.

File: mvg/mvg.py [hunk 2]
```diff
@@ -396,7 +396,7 @@ def create_measurement(
 
     # in example
     def read_measurements(self, sid: str):
```
Comment (tuix) [inline mvg/mvg.py:398]: To be consist with similar functions in the package, `list_sources()` and `list_analyses()`, I suggest we rename this function to `list_measurements()`

**Review by tuix — CHANGES_REQUESTED** [review]

File: mvg/mvg.py (hunk 1 above)
Comment (vnadhan) [reply mvg/mvg.py:399]: @SergioMDCB Good point. I assume you mean "Would be a good idea ..."

**Comment by vnadhan** [comment]
@tuix @SergioMDCB 
I have not updated the docs. I will update the docs in another PR when a new release for MVG is done.

**Comment by SergioMDCB** [comment]
No worries, we need to update the overall documentation once we incorporate the other PR with the other changes and the introduction to the client side of the emerging modes feature

Sergio Martin del Campo
Data Scientist – Viking Analytics
Phone: +46 (0)73 829 9847
***@***.******@***.***>
www.vikinganalytics.se<http://www.vikinganalytics.se/>
c/o HiQ, Östra Hamngatan 24, 411 09 Gothenburg, Sweden

**Review by SergioMDCB — APPROVED** [review]
This PR renames the read_measurements function to list_measurements and provides with the metadata of the measurements only instead of the full data

**Review by tuix — APPROVED** [review]

**Pull request merged**